admin/src/js/controllers/export-feedback.js

```diff
@@ -9,11 +9,15 @@ angular.module('controllers').controller('ExportFeedbackCtrl',
     'use strict';
     'ngInject';
 
+    var MESSAGE_LIMIT = 120;
+
     var safeStringify = function(str) {
       if (typeof str === 'string') {
         return str;
       }
       try {
+        var suffix = str.message.length > MESSAGE_LIMIT ? '...' : '';
```

Comment:
Will this cover all cases? It looks like line 16 above is not going to be limited?